contracts/VaultV3.vy

```diff
@@ -942,10 +954,12 @@ def shutdown_vault():
 ## ERC20 + ERC4626 ##
 @external
 def deposit(assets: uint256, receiver: address) -> uint256:
+    self.is_whitelisted(receiver)
```

Comment:
this allows a whitelisted msg.sender to swap addresses and deposit into the vault no?

Reply:
same as below